site/pages/create-disruption.tsx

```diff
@@ -1,9 +1,138 @@
-const CreateDisruption = () => {
-    return <></>;
+import React, { ReactElement } from "react";
+import DateSelector from "components/DateSelector";
+import TimeSelector from "components/TimeSelector";
+import { BaseLayout } from "components/layout/Layout"
+import { DisruptionValidity } from "interfaces";
+
+const title = 'Create Disruptions';
+const description = 'Create Disruptions page for the Create Transport Disruptions Service';
+
+interface CreateDisruptionProps {
+    inputs: DisruptionValidity
+}
+
+const CreateDisruption = ({ inputs }: CreateDisruptionProps): ReactElement => {
+    return (
+        <BaseLayout title={title} description={description}>
+            <>
+                <div className="govuk-form-group">
+                    <fieldset className="govuk-fieldset" role="group" aria-describedby="when-is-the-disruption">
```

Comment:
This aria-describedby isn't referencing anything so can be removed

Reply:
done